src/components/safe-apps/AppFrame/useAppCommunicator.ts

```diff
@@ -32,22 +36,26 @@ type JsonRpcResponse = {
   error?: string
 }
 
-type UseAppCommunicatorConfig = {
-  app?: SafeAppData
+type UseAppCommunicatorHandlers = {
```

Comment:
Changing this a little bit thinking in a future separation for this hook to a separate package. Now mostly of the dependences are coming from the SDK so should be easier